src/style.css

```diff
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+/* @font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+} */
+
+body {
+  font-weight: 300;
+}
+
+.form_field1 {
+  position: relative;
+  height: 68px;
+  border: 1px solid #e1e7ed;
+  margin-top: 20px;
+  border-radius: 4px;
+  background: #fff;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.form_field2 {
+  position: relative;
+  height: 40px;
+  border: 1px solid #e1e7ed;
+  margin-top: 20px;
+  border-radius: 4px;
+  background: #fff;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
```

Comment:
you have only `height` style difference
create 1 class for both form fields, and additional to control height